app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1>Contestants</h1>
+<% @contestants.each do |contestant| %>
+  <h3><%= contestant.name %></h3>
+  <p>Projects: <%= contestant.project_names %></p>
```

Comment:
It looks like you were working through this view before time might have run out on you. I do want to mention, however, that because we are following a TDD approach, you want to make sure your tests are fully fleshed out before starting on the code that implements the feature you're testing.